galactic/release-rhel-build.yaml

```diff
@@ -37,6 +37,7 @@ package_blacklist:
 - marti_visualization_msgs  # Requires release with bloom >= 0.10.2
 - mrpt2  # ffmpeg has no RPM package in supported repositories
 - ntpd_driver  # libpoco-dev has no RPM package for RHEL 8
+- octomap  # libqglviewer-dev-qt5 has no RPM for RHEL
```

Comment:
Thanks for the swift action - can we perhaps only block `octovis` rather than `octomap`? Or will it have to be the entire repository?

Reply:
Bloom generates spec files on a per-repository basis, and we don't currently have a mechanism to skip a single package in the repository on a per-platform basis. Right now, any missing dependencies in any packages in a repository means that none of the packages in the repository get generated.

Reply:
Thanks for the insights and for the swift action :-)